models/product-discount-value-relative/src/product-discount-value-relative-draft/builder.spec.ts

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable jest/no-disabled-tests */
+/* eslint-disable jest/valid-title */
```

Comment:
🦾

Reply:
We do need these, my linter complains if they're omitted